src/lib/types.ts

```diff
@@ -0,0 +1,6 @@
+import { KualiSubject } from './fetchers';
+
+export interface KualiSubjectInSession {
+  inSession?: boolean;
+}
+export type InSessionSubject = KualiSubject & KualiSubjectInSession;
```

Comment:
This can be combined into one type declaration. Also, I would suggest not calling it `InSessionSubject` as we might use it for other stuff :)